R/pkgnet.R

```diff
@@ -33,14 +33,66 @@ globalVariables(c('.'
                   , 'SYMBOL'
                   ))
 
-# NULL object for common parameter documentation
-#' @param pkg_name (string) name of a package
-#' @param pkg_path (string) The path to the package repository. If given, coverage
-#'                 will be calculated for each function. \code{pkg_path} can be an
-#'                 absolute or relative path.
-#' @name doc_shared
-#' @title NULL Object For Common Documentation
-#' @description This is a NULL object with documentation so that later functions can call
-#' inheritParams
-NULL
 
+#' @title pkgnet : Network Analysis of R Packages
```

Comment:
@jayqi what does this section of code do for the documentation?

Reply:
This makes a Roxygen article that shows up if you do `?pkgnet` or `?pkgnet-package`

<img width="500" alt="Screen Shot 2019-03-11 at 8 32 54 PM" src="https://user-images.githubusercontent.com/2721979/54168614-646bf000-443d-11e9-9aa4-fc55a2020408.png">